src/style.css

```diff
@@ -1 +1,22 @@
 /* styles go here */
+.fieldset:not(:first-child) {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.form-field {
+  display: block;
+  margin: 10px 0;
+}
+
+.form-field-top {
+  margin-top: 0;
+}
+
+.form-field-bot {
+  margin-bottom: 0;
+}
+
```

Comment:
Let's add only the margin-bottom and using :last-child remove it from the last labels